### ISSUE TITLE
Removing Text from Images using CV2 and Keras-OCR 

### ISSUE THREAD

**Issue by aj120898** [tensorflow/tensorflow#57621]
<details><summary>Click to expand!</summary> 
 
 ### Issue Type

Support

### Source

source

### Tensorflow Version

2.8.0

### Custom Code

No

### OS Platform and Distribution

windows

### Python version

3.10

### Current Behaviour?

```shell
Removing Text from Images using CV2 and Keras-OCR -while removing text from image it's creating blurriness in background for some images.
```

### Standalone code to reproduce the issue

```shell
will send soon in comment
```

### Relevant log output

_No response_</details>

**Comment by mohantym** [comment]
Hi @aj120898 !
Could you provide a minimal stand alone code to replicate the issue.
Thank you!

**Comment by aj120898** [comment]
below code i am using to remove existing text and then for adding new text to the image ( its creating blurriness in the background of the text )


from tkinter import filedialog
from tkinter import simpledialog
import os # to interact with operating system
import tkinter as tk
from tkinter import *
from tkinter import messagebox,Label
from PIL import Image,ImageTk,ImageDraw
import glob
import pytesseract
import string
import random
import keras_ocr
import cv2
import math
import numpy as np


root = tk.Tk()
root.withdraw()
newval=1

user_inp1= simpledialog.askstring(title="Augmentor",prompt="Please enter image type structured or unstructured")
if(user_inp1=="structured" or user_inp1=="Structured" ):
    pass
elif(user_inp1=="unstructured" or user_inp1=="Unstructured" ):
    import unstructured_imaugmentor
else:
    messagebox.showinfo("Information","Please enter proper numeric value greather than 1 ")
    DestroyExit()
def showimage():
        messagebox.showinfo("Information","Browse 1 Image")
        filename=filedialog.askopenfilename(title="select image file",filetype=(("JPG File","*.jpg"),("PNG File","*.png"),("JPEG File","*.jpeg"),("TIFF File","*.tiff"),("All file","*.*")))
        img=Image.open(filename)
        a=a="D:\\number_plate\\no_plates_images\\"+ str(x) + ".png"
        print(a)
        img.save(a)
def exit():
    entry=messagebox.askyesnocancel("Oops","Are you sure you want to exit?")
    if entry==True:
        wind.destroy()
def DestroyExit():
        wind.destroy()
for x in range(newval):
    if(x==newval):
        DestroyExit()
        DestroyExit()
    else:
        showimage()
        messagebox.showinfo("Information","Image is Saved")
root = tk.Tk()
root.withdraw()
user_inp= simpledialog.askstring(title="Image Process",prompt="How many images you want to generate")
if not user_inp.isnumeric() or int(user_inp) < 1:
    messagebox.showinfo("Information","Please enter proper numeric value greather than 1 ")
    DestroyExit()
else:
    newval=int(user_inp)  
inputcount=newval
pytesseract.pytesseract.tesseract_cmd = r"C:\\Users\\jahanvi.gupta\\AppData\\Local\\Programs\\Tesseract-OCR\\tesseract.exe"

image = Image.open("D:\\number_plate\\no_plates_images\\"+ str(x) + ".png")
image_to_text = pytesseract.image_to_string(image, lang='eng')
print(image_to_text)
inputtext=image_to_text
inputtext=inputtext.rstrip()
inputlength=len(inputtext)

i=0
outputstring=""
augmented_text=[]
while(i<inputcount):
    outputstring=""
    i=i+1
    for f in inputtext:
        if(f.isalpha()):
                if(f.isupper()):
                    a=random.choice(string.ascii_uppercase)
                    a=str(a)
                    outputstring=outputstring+a
                else:
                    a=random.choice(string.ascii_lowercase)
                    a=str(a)
                    outputstring=outputstring+a
        elif(f.isnumeric()):
                b=random.randint(0,9)
                b=str(b)
                outputstring=outputstring+b
        else:
            outputstring=outputstring+f
            outputstring.rstrip()
    augmented_text.append(outputstring)
b="D:\\number_plate\\no_plates_images\\"+ str(x) + ".png"

**Comment by mohantym** [comment]
Hi @aj120898 !
I don't see much usage of Tensorflow/Keras in above code . Rather Tesseract.
Attaching the documentation of [Keras-OCR](https://keras-ocr.readthedocs.io/en/latest/) and sample [usage notebook](https://colab.sandbox.google.com/gist/mohantym/e7e91a5de49178c5ed7f8debaba4ecbd/keras-ocr-notebook.ipynb) for your reference.

Please post on TF forum for further assistance.

Thank you!
